PLATFORM_LIBS.md

```diff
@@ -36,6 +36,8 @@ cross-compilation targets.  `Kotlin/Native` distribution provides access to
 `OpenGL`, `zlib` and other popular native libraries on
 applicable platforms.
 
+> Note : Some libraries are not included with Kotlin on all operating systems, for example OpenGL is not available directly with Kotlin on Windows.
```

Comment:
OpenGL is available (as `platform.opengl32`) on Windows.

Reply:
Okay, so sorry for that.